src/main/scala/com/codacy/client/bitbucket/service/PullRequestServices.scala

```diff
@@ -92,4 +94,10 @@ class PullRequestServices(client: BitbucketClient) {
     client.delete(url)
   }
 
+  def listComments(author: String, repo: String, pullRequestId: Int): RequestResponse[Seq[SimplePullRequestComment]] = {
+    val url = s"https://bitbucket.org/!api/1.0/repositories/$author/$repo/pullrequests/$pullRequestId/comments"
```

Comment:
The pagination is for v2, and unfortunately v2 no longer has the "anchor" that I need.. Even bitbucket themselves use their v1 API for comments when showing a PR online 😭